helm/libcoder/templates/_coder.yaml

```diff
@@ -43,6 +43,10 @@ spec:
       nodeSelector:
       {{ toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.coder.topologySpreadConstraints }}
+      topologySpreadConstraints:
+      {{- toYaml .Values.coder.topologySpreadConstraints | nindent 8 }}
+      {{- end }}
```

Comment:
A `with X` block sets `.` equal to `X` within that block. 

c.f.

```
      {{- with .Values.coder.affinity }}
      affinity:
      {{- toYaml . | nindent 8 }}
      {{- end }}
```

It only refers to `.Values.coder.affinity` once, in the `with` clause. Then, we use `.` to refer to it when calling `toYaml`

Reply:
ah makes sense - although adding that does not fix the issue. i'll push a commit with this change for posterity.

Reply:
Worked for me --- how are you testing?